docs/src/modules/java/pages/actions-as-controller.adoc

```diff
@@ -36,14 +36,14 @@ request and only conditionally forward the request to the entity if the verifica
 ----
 include::example$java-spring-valueentity-shopping-cart/src/main/java/com/example/api/ShoppingCartController.java[tag=forward]
 ----
-<1> `KalixClient` is injected on the constructor. It will be used to build calls to the underlining Entity.
+<1> `ComponentClient` is injected on the constructor. It will be used to build calls to the underlining Entity.
 <2> Expose the command handler as a `POST` endpoint at specified path.
 <3> Check if the added item is carrots.
 <4> If it is "carrots" immediately return an error, disallowing adding the item.
-<5> For allowed requests, use `kalixClient` to get a deferred call to the entity.
+<5> For allowed requests, use `componentClient` to get a deferred call to the entity.
 <6> The `deferredCall` is then used with `effects().forward()` to forward the request to the entity.
 
-NOTE: You might be wondering what the `kalixClient` is about. For now, think of it as a lightweight HTTP client allowing you to reach out to other Kalix services. All details can be found at xref:call-another-service.adoc[] chapter.
+NOTE: You might be wondering what the `componentClient` is about. For now, think of it as a lightweight, type safe, HTTP client allowing you to reach out to other Kalix services. All details can be found at xref:call-another-service.adoc[] chapter.
```

Comment:
note this file `actions-as-controller.adoc` was deleted in main with the menu restructure. Now this content is under `actions.adoc´.

Reply:
PR updated